.travis.yml

```diff
@@ -0,0 +1,49 @@
+# This is the simple Travis configuration, which is intended for use
+# on applications which do not require cross-platform and
+# multiple-GHC-version support. For more information and other
+# options, see:
+#
+# https://docs.haskellstack.org/en/stable/travis_ci/
+#
+# Copy these contents into the root directory of your Github project in a file
+# named .travis.yml
+
+# Choose a build environment
+dist: xenial
+
+# Do not choose a language; we provide our own build tools.
+language: generic
+
+services:
+  - postgresql
+
+# Caching so the next build will be fast too.
+cache:
+  directories:
+  - $HOME/.stack
+  - .stack-work
+
+# Ensure necessary system libraries are present
+addons:
+  apt:
+    packages:
+      - libgmp-dev
+
+before_install:
+# Download and unpack the stack executable
+- mkdir -p ~/.local/bin
+- export PATH=$HOME/.local/bin:$PATH
+- wget -qO- https://get.haskellstack.org/ | sh
+
+install:
+# Build dependencies
+- stack --no-terminal --install-ghc test --only-dependencies
+
+script:
+# Build the package, its tests, and its docs and run the tests
+- stack --no-terminal build --fast
+- psql -c 'CREATE DATABASE testsupplychainserver;' -U postgres
+- psql -c 'CREATE DATABASE testbusinessregistry;' -U postgres
+- stack exec supplyChainServer -- --init-db -c 'dbname=testsupplychainserver'
+- echo 'YES' | stack exec businessRegistry -- initdb -c 'dbname=testbusinessregistry'
```

Comment:
A little bit outside the scope of a PR review... but what does echoing 'YES' into this command do? I've not seen it wait for a YES/NO in the other builds - Is this something relatively new....?

Reply:
In the Business Registry, when running with the option `initdb`, it shows you the database migration and asks you to type in 'YES' if you are happy with that. You may have been running with the `server` option, which doesn't have this.

Reply:
Looked into it, My build was silently failing. Seems to be a docker-compose issue though at first glance.